packages/block-library/src/social-link/index.php

```diff
@@ -232,7 +232,7 @@ function block_core_social_link_services( $service = '', $field = '' ) {
+			'icon' => '<svg width="24" height="24" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false"><path d="M22,12.0681004 C22,10.8494624 21.0143369,9.88172043 19.8136201,9.88172043 C19.2222222,9.88172043 18.6845878,10.1146953 18.2903226,10.4910394 C16.7849462,9.41577061 14.7240143,8.71684588 12.4301075,8.62724014 L13.4336918,3.93189964 L16.6953405,4.63082437 C16.7311828,5.45519713 17.4121864,6.11827957 18.2544803,6.11827957 C19.1146953,6.11827957 19.8136201,5.41935484 19.8136201,4.55913978 C19.8136201,3.69892473 19.1146953,3 18.2544803,3 C17.6451613,3 17.1075269,3.35842294 16.8566308,3.8781362 L13.218638,3.10752688 C13.1111111,3.08960573 13.0035842,3.10752688 12.9318996,3.16129032 C12.8422939,3.21505376 12.7885305,3.3046595 12.7706093,3.41218638 L11.6594982,8.64516129 C9.3297491,8.71684588 7.23297491,9.41577061 5.70967742,10.5089606 C5.31541219,10.1326165 4.77777778,9.89964158 4.18637993,9.89964158 C2.96774194,9.89964158 2,10.8853047 2,12.0860215 C2,12.9820789 2.53763441,13.734767 3.29032258,14.0752688 C3.25448029,14.2903226 3.23655914,14.5053763 3.23655914,14.7383513 C3.23655914,18.1075269 7.16129032,20.8494624 12,20.8494624 C16.8387097,20.8494624 20.7634409,18.125448 20.7634409,14.7383513 C20.7634409,14.5232975 20.7455197,14.2903226 20.7096774,14.0752688 C21.4623656,13.734767 22,12.9641577 22,12.0681004 Z M6.98207885,13.6272401 C6.98207885,12.7670251 7.68100358,12.0681004 8.54121864,12.0681004 C9.40143369,12.0681004 10.1003584,12.7670251 10.1003584,13.6272401 C10.1003584,14.4874552 9.40143369,15.1863799 8.54121864,15.1863799 C7.68100358,15.2043011 6.98207885,14.4874552 6.98207885,13.6272401 Z M15.7096774,17.7670251 C14.6344086,18.8422939 12.5913978,18.9139785 12,18.9139785 C11.390681,18.9139785 9.34767025,18.8243728 8.29032258,17.7670251 C8.12903226,17.6057348 8.12903226,17.3548387 8.29032258,17.1935484 C8.4516129,17.0322581 8.70250896,17.0322581 8.86379928,17.1935484 C9.54480287,17.874552 10.9784946,18.1075269 12,18.1075269 C13.0215054,18.1075269 14.4731183,17.874552 15.1362007,17.1935484 C15.297491,17.0322581 15.5483871,17.0322581 15.7096774,17.1935484 C15.8530466,17.3548387 15.8530466,17.6057348 15.7096774,17.7670251 Z M15.4229391,15.2043011 C14.562724,15.2043011 13.8637993,14.5053763 13.8637993,13.6451613 C13.8637993,12.7849462 14.562724,12.0860215 15.4229391,12.0860215 C16.2831541,12.0860215 16.9820789,12.7849462 16.9820789,13.6451613 C16.9820789,14.4874552 16.2831541,15.2043011 15.4229391,15.2043011 Z"></path></svg>',
```

Comment:
There are quite a few decimal places on this SVG. I ran the SVG through https://jakearchibald.github.io/svgomg/ and reduced the complexity a bit down from 2500 bytes to 970 bytes. I know, peanuts, but here's the new file:

```
<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" aria-hidden="true"><path d="M22 12.068a2.184 2.184 0 0 0-2.186-2.186c-.592 0-1.13.233-1.524.609-1.505-1.075-3.566-1.774-5.86-1.864l1.004-4.695 3.261.699A1.56 1.56 0 1 0 18.255 3c-.61-.001-1.147.357-1.398.877l-3.638-.77a.382.382 0 0 0-.287.053.348.348 0 0 0-.161.251l-1.112 5.233c-2.33.072-4.426.77-5.95 1.864a2.201 2.201 0 0 0-1.523-.61 2.184 2.184 0 0 0-.896 4.176c-.036.215-.053.43-.053.663 0 3.37 3.924 6.111 8.763 6.111s8.763-2.724 8.763-6.11c0-.216-.017-.449-.053-.664A2.207 2.207 0 0 0 22 12.068Zm-15.018 1.56a1.56 1.56 0 0 1 3.118 0c0 .86-.699 1.558-1.559 1.558-.86.018-1.559-.699-1.559-1.559Zm8.728 4.139c-1.076 1.075-3.119 1.147-3.71 1.147-.61 0-2.652-.09-3.71-1.147a.4.4 0 0 1 0-.573.4.4 0 0 1 .574 0c.68.68 2.114.914 3.136.914 1.022 0 2.473-.233 3.136-.914a.4.4 0 0 1 .574 0 .436.436 0 0 1 0 .573Zm-.287-2.563a1.56 1.56 0 0 1 0-3.118c.86 0 1.56.699 1.56 1.56 0 .841-.7 1.558-1.56 1.558Z"/></svg>
```